addons/mail/static/src/components/link_preview_image_view/link_preview_image_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.LinkPreviewImageView" owl="1">
+        <div class="o_LinkPreviewImageView position-relative me-3 mb-2" t-attf-class="{{ className }}" t-on-mouseenter="linkPreviewImageView.onMouseEnter" t-on-mouseleave="linkPreviewImageView.onMouseLeave" t-ref="root">
+            <a t-att-href="linkPreviewImageView.image" target="_blank">
+                <img t-if="linkPreviewImageView.image" class="img-fluid rounded" t-att-src="linkPreviewImageView.image" t-att-alt="linkPreviewImageView.image"/>
```

Comment:
field image should be named imageUrl, and alt might as well stay empty if it contains the url itself? maybe use preview title if there is one?

Reply:
This case is only when you add an url to an image, so you will never have any kind of title.
Better remove the alt for this I guess.